src/lib/middleware/errorHandler.tsx

```diff
@@ -69,6 +70,12 @@ export const errorHandlerMiddleware = async (
     console.log(detail)
   }
 
+  // Log server errors (code, stack trace) when in production mode
+  if (NODE_ENV === "production" && code >= 500) {
```

Comment:
There's a guard in [`logger.ts`](https://github.com/artsy/force/blob/main/src/v2/Utils/logger.ts#L20) that will only dispatch if in prod, so no need to check ENV here